src/io.jl

```diff
@@ -101,7 +101,7 @@ function Base.show(io::IO, t::Transition)
 end
 
 function Base.show(io::IO, zdt::ZonedDateTime)
-    if get(io, :compact, false) || get(io, :compact_el, false)
+    if VERSION <= v"1.5.0-DEV.163" && get(io, :compact_el, false)
```

Comment:
don't we still want to check for `:compact` here?

Reply:
I don't think so? See the output example: https://github.com/JuliaTime/TimeZones.jl/pull/247#issuecomment-582023513

Reply:
If we're running julia <= v"1.5.0-DEV.163"
then `show(IOContext(stdout, :compact=>true), d)`  would end up printing the non-compact version here without that check that was removed no?

Reply:
This is also true for `TimeType`s on Julia 1.5-DEV.

```julia
julia> show(IOContext(stdout, :compact=>true), Date(2000))
Date("2000-01-01")
```

I'll note that this PR is trying to mirror the 1.5 behaviour on all versions of Julia

Reply:
ahh ok. that makes sense.